docs/root/configuration/observability/stat_sinks/open_telemetry_stat_sink.rst

```diff
@@ -0,0 +1,12 @@
+.. _config_stat_sinks_open_telemetry:
+
+OpenTelemetry Stat Sink
+=========================
+
+The :ref:`OpenTelemetryStatsSink <envoy_v3_api_msg_extensions.stat_sinks.open_telemetry.v3.SinkConfig>` configuration specifies a
+stat sink that emits stats according to `OpenTelemetry Protocol Specification <https://opentelemetry.io/docs/reference/specification/protocol/otlp/>`.
```

Comment:
IIRC this is not sphinx external URL format but I can't remember. Please check the rendered docs.